### DISCUSSION TITLE
Get data grouped from handleSubmit

### DISCUSSION THREAD

**Discussion by FilipBrannlund** [react-hook-form/react-hook-form#5141] [Q&A] [answered]
Is it possible to recieve data grouped from handleSubmit function?
When having multiple group of fields the data from handleSubmit can be complex

See example here (console.log): https://codesandbox.io/s/mystifying-rgb-wvzly?file=/src/App.js

handleSubmit delivers following data:
```
firstName0: ""
lastName0: ""
firstName1: ""
lastName1: ""
firstName2: ""
lastName2: ""
firstName3: ""
lastName3: ""
firstName4: ""
lastName4: ""
firstName5: ""
lastName5: ""
firstName6: ""
lastName6: ""
firstName7: ""
lastName7: ""
```

Is it possible to recieve data grouped? 
for example
```
person0: {
    firstName: ""
    lastName: ""
},
person1: {
    firstName: ""
    lastName: ""
},
person2: {
    firstName: ""
    lastName: ""
}
person3: {
    firstName: ""
    lastName: ""
}
```

**Comment by bluebill1049** [comment, marked as answer]
eg: `<inpput {...register('person.0.firstName')}/>`
